recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ outputs:
       run:
         - cinnabar ~=0.5.0
         - click >=8.2.0,<9.0
-        - gufe ==1.7.0
+        - gufe >=1.7.1,<1.8
```

Comment:
```suggestion
        - gufe ~=1.7.0
```

Reply:
we don't want it to pull in 1.7.0 though, right?

Reply:
are they incompatible? I thought we just needed it relaxed